content_scripts/vimium_frontend.coffee

```diff
@@ -705,8 +484,8 @@ root.Frame = Frame
 root.windowIsFocused = windowIsFocused
 root.bgLog = bgLog
 # These are exported for find mode and link-hints mode.
-extend root, {handleEscapeForFindMode, handleEnterForFindMode, performFind, performBackwardsFind,
-  enterFindMode, focusThisFrame}
+extend root, {focusFoundLink, selectFoundInputElement, focusThisFrame, FocusSelector,
```

Comment:
@mrmr1993 Might it not be better to move these to `mode_normal.coffee` too?

They only exist to serve normal mode, do they not?

Reply:
`focusFoundLink` and `selectFoundInputElement` are now only used in `mode_find.coffee`, so they could go there.

`focusThisFrame` is also used by link hints. `FocusSelector` does a similar job to other focus trackers, so I was planning to try and merge them into a kind of `FocusOracle` that other code calls out to.